kdlc/cli.py

```diff
@@ -22,7 +21,29 @@
     help="The input file, either .knwf or .kdl",
     type=click.Path(exists=True),
 )
-def prompt(input_file: str, output_file: str) -> None:
+@click.option(
+    "--debug",
+    "-d",
+    "debug_logging",
+    required=False,
+    help="Print debug logging to stdout",
+    is_flag=True,
+)
+def prompt(input_file: str, output_file: str, debug_logging) -> None:
+    logger.remove()
+    if debug_logging:
+        logger.add(
+            sys.stdout,
+            level="DEBUG",
+            format="<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> - <level>{message}</level>",
+        )
+    else:
+        logger.add(
+            sys.stderr,
+            level="ERROR",
+            format="<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> - <level>{message}</level>",
```

Comment:
Maybe abstract this string out into a variable.  Seems like it's the same on lines 38 and 44.

Reply:
Actually... looking at this again, I think we might just need the if condition.  It seems there isn't a use of logger.error, so I'd probably remove it for now.

Reply:
Resolved: https://github.com/k-descriptor-language/kdl/commit/fefb4be17976d72bed0d87a9f9ba1401878c084f